src/app/components/cds/charts/NumberWidget.js

```diff
@@ -13,11 +13,9 @@ const NumberWidget = ({
     <div className={styles.numberWidgetTitle}>
       {title}
     </div>
-    { itemCount &&
-      <div className={styles.numberWidgetItemCount}>
-        {itemCount} { unit && itemCount !== '-' && <span className={styles.numberWidgetUnit}>{unit}</span> }
-      </div>
-    }
+    <div className={styles.numberWidgetItemCount}>
+      { itemCount || '-' } { unit && itemCount !== '-' && itemCount !== null && <span className={styles.numberWidgetUnit}>{unit}</span> }
```

Comment:
Same here, default prop `itemCount` could be `null` and this line would read `{ itemCount || '-' } { itemCount && unit && <span bla bla...`